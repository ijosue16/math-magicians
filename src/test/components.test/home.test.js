import React from "react";
import renderer from "react-test-renderer";
import Home from "../components/home";


describe("render home page", () =>{
    test("to match snapshot",() =>{
        const tree=renderer.create(<Home/>).toJSON();
        expect(tree).toMatchSnapshot();
    });
})