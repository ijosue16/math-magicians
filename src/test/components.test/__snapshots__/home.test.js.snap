// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render home page to match snapshot 1`] = `
<div
  className=" m-0 py-5 d-flex flex-column py-5 home justify-content-center container-fluid "
>
  <h4
    className=" m-0 w-100 p-3 pb-5"
  >
    Welcome to our page!
  </h4>
  <p
    className="px-3"
  >
    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Lectus magna fringilla urna porttitor rhoncus dolor. Metus dictum at tempor commodo ullamcorper a lacus vestibulum sed. Arcu odio ut sem nulla pharetra diam sit amet. Id leo in vitae turpis massa sed elementum tempus egestas. Non quam lacus suspendisse faucibus. Nec nam aliquam sem et tortor consequat. Aliquam sem fringilla ut morbi tincidunt augue interdum. Id cursus metus aliquam eleifend mi. Eu feugiat pretium nibh ipsum consequat nisl vel. Urna neque viverra justo nec ultrices dui sapien eget. Nullam ac tortor vitae purus faucibus ornare suspendisse sed nisi. Eget dolor morbi non arcu risus quis varius.
    <br />
    <br />
    Dui vivamus arcu felis bibendum ut tristique. Ultricies integer quis auctor elit sed vulputate mi sit. Proin nibh nisl condimentum id venenatis a. Orci eu lobortis elementum nibh. Fermentum odio eu feugiat pretium nibh. Feugiat vivamus at augue eget arcu dictum varius duis. Quam viverra orci sagittis eu volutpat odio facilisis mauris sit. Et netus et malesuada fames ac turpis egestas integer. Vitae purus faucibus ornare suspendisse sed. Arcu vitae elementum curabitur vitae nunc sed velit. Ultrices eros in cursus turpis. Tincidunt tortor aliquam nulla facilisi. Morbi tristique senectus et netus. Viverra accumsan in nisl nisi scelerisque eu ultrices. Pulvinar neque laoreet suspendisse interdum consectetur libero. Placerat orci nulla pellentesque dignissim. Rhoncus aenean vel elit scelerisque mauris pellentesque.
  </p>
</div>
`;
